java/src/main/java/org/maplibre/mlt/converter/encodings/GeometryEncoder.java

```diff
@@ -624,11 +624,21 @@ public static EncodedGeometryColumn encodeGeometryColumn(
 
     // TODO: get rid of that separate calculation
     var minVertexValue =
-        Collections.min(
-            vertexBuffer.stream().flatMapToInt(v -> IntStream.of(v.x(), v.y())).boxed().toList());
+        vertexBuffer.isEmpty()
```

Comment:
This should no longer be necessary, as layers with no features are skipped at a higher level.